jetty-websocket/websocket-core-common/src/main/java/org/eclipse/jetty/websocket/core/internal/util/AbstractJettyMethodHandle.java

```diff
@@ -0,0 +1,55 @@
+//
+// ========================================================================
+// Copyright (c) 1995-2022 Mort Bay Consulting Pty Ltd and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+// ========================================================================
+//
+
+package org.eclipse.jetty.websocket.core.internal.util;
+
+import java.lang.invoke.MethodHandle;
+
+public abstract class AbstractJettyMethodHandle implements JettyMethodHandle
```

Comment:
Does this class provide any value that you couldn't get with default impls in the interace?

Reply:
I have removed this abstract class and put default methods in the interface.